### ISSUE TITLE
How did you train vggface model, what parameters did you use?

### ISSUE THREAD

**Issue by marinkreso95** [rcmalli/keras-vggface#60]
I would like to know how did you train model whose weights are loaded into chosen model (Resnet, Senet). 

Did you train the model in keras, how you implemented triplet loss, what training parameters did you use (iterations, learning rate etc.) to get amazing results?

The information that I asked about above is for Resnet50 architecture.

**Comment by Servon-Lee** [comment]
Hi, there. Do u figure out those parameters? Why the loss(softmax) does not reduce when I use the pre-trained model?

**Comment by sherlockchou86** [comment]
I guess there is no triplet loss used in training the model. Just use softmax loss for classification and then we can remove top layers to use features for generic usage such as fine-tuning.

**Comment by ritikjain51** [comment]
For the training of the model, I guess they have used multi-class cross-entropy loss. But not able to understand why have used softmax activation on the last layer.
